.jenkins/Jenkinsfile

```diff
@@ -203,16 +203,69 @@ pipeline {
             }
           }
         }
-        stage('Windows Debug') {
-           agent {
-             node {
-               label 'SGXFLC-Windows'
-             }
-
-           }
-           steps {
-             bat '''mkdir build && cd build && cmake -G "Visual Studio 15 2017 Win64" .. && pushd . && "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\Common7\\Tools\\LaunchDevCmd.bat" && popd && cmake --build . --config Debug && ctest -C Debug'''
-           }
+        stage('Windows Debug Crossplat') {
+          stages {
+            stage('Linux SGX1 Debug') {
+              agent {
+                docker {
+                  image 'oetools-azure:1.6'
+                }
+
+              }
+              steps {
+                timeout(15) {
+                  sh './scripts/test-build-config -p SGX1FLC -b Debug --compiler=clang-7'
+                  stash includes: 'build/tests/**', name: 'linuxdebug'
+                }
+              }
+            }
+
+            stage('Windows Debug') {
+              agent {
+                node {
+                  label 'SGXFLC-Windows'
+                }
+              }
+
+              steps {
+                unstash 'linuxdebug'
+                bat 'move build linuxbin'
+                bat 'mkdir build && cd build && cmake -G "Visual Studio 15 2017 Win64" -DADD_WINDOWS_ENCLAVE_TESTS=1 -DLINUX_BIN_DIR=%cd%\\linuxbin\\tests .. && pushd . && "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\Common7\\Tools\\LaunchDevCmd.bat" && popd && cmake --build . --config Debug && ctest -V -C Debug'
```

Comment:
Again, consider breaking this across lines for readability, or pushing this into a separate script file.

Reply:
@CodeMonkeyLeet Thanks Simon, I'll add a separate script file in my next PR